hotham/src/shaders/pbr.frag

```diff
@@ -286,27 +267,27 @@ void main()
 	// Calculation of analytical lighting contribution
 	vec3 diffuseContrib = (1.0 - F) * diffuse(pbrInputs);
 	vec3 specContrib = F * G * D / (4.0 * NdotL * NdotV);
+
 	// Obtain final intensity as reflectance (BRDF) scaled by the energy of the light (cosine law)
 	vec3 color = NdotL * (diffuseContrib + specContrib);
 
-	// Calculate lighting contribution from image based lighting source (IBL)
-	color += getIBLContribution(pbrInputs, n, reflection) * DEFAULT_IBL_SCALE;
+	// Calculate lighting contribution from image based lighting source (IBL), scaled by a scene data parameter.
+	color += getIBLContribution(pbrInputs, n, reflection) * sceneData.params.x;
```

Comment:
Could be nice to avoid calling this function if `sceneData.params.x` is zero. Not sure how good that works in a shader though.

Reply:
Yep, and this is fixed in #314